src/element.ts

```diff
@@ -141,12 +141,26 @@ export abstract class Element {
    */
   setStyle(style: ElementStyle | undefined): this {
     this.style = style;
+    const svg = this.getSVGElement();
+    if (svg) {
+      if (style?.fillStyle) {
+        svg.setAttribute('fill', style.fillStyle);
+        for (let i = 0; i < svg.children.length; i++) svg.children[i].setAttribute('fill', '');
+      }
+    }
     return this;
   }
 
   /** Set the element & associated children style used for rendering. */
   setGroupStyle(style: ElementStyle): this {
     this.style = style;
+    const svg = this.getSVGElement();
+    if (svg) {
+      if (style?.fillStyle) {
+        svg.setAttribute('fill', style.fillStyle);
+        for (let i = 0; i < svg.children.length; i++) svg.children[i].setAttribute('fill', '');
```

Comment:
Same here. I think having extra braces around the single line of code is nicer, in case we want to add more lines of code someday. It also makes it easier for me to scan the code quickly (especially during code review).

Reply:
the for statement is no longer required after #1457, removed